source/_docs/z-wave/controllers.markdown

```diff
@@ -63,6 +63,19 @@ The alternative to a stick is a hub that supports Z-Wave. Home Assistant support
 
 ## Controller Notes
 
+### 800 Series Controllers
+
+Some features are not supported with 800 series controllers yet.
+Unsupported
+ - Long Range
```

Comment:
There are also 700 series controllers that support Long Range. Z-Wave JS does not. Seems weird to mention it here.